hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -286,12 +286,46 @@ sudo reboot
 ## USB mass storage boot
 This is only available in the BETA release and requires updated (not yet released) firmware via rpi-update. If you aren't already familiar with how to use a USB drive for the root filesystem then you probably want to wait until this feature is in the default Raspbian image.
 
+There's no support for migrating a SD card image to a USB drive. It's possible but the process can potentially be quite involved and varies according to your original setup. Please see [STICKY: HOWTO: Move the filesystem to a USB stick/Drive](https://www.raspberrypi.org/forums/viewtopic.php?f=29&t=44177&start=350)
+
+## BETA setup instructions
+These instructions assume that you are familiar with manual firmware and bootloader updates and understand how to revert to a previous version if you want to revert the changes. If not, please wait until the features are available in a full Raspbian release image.
+
+There will be a post on the Raspberry Pi [General Discussion forum](https://www.raspberrypi.org/forums/viewforum.php?f=63) when the binaries are availble.
+
+### Update the bootloader
+```
+# From a standard Raspbian SD card boot
+sudo apt update
+sudo apt dist-upgrade
+
+# As root edit /etc/default/rpi-eeprom-update and select BETA releases
+
+# Install the BETA bootloader and replace current configuration settings to enable USB boot.
+sudo rpi-eeprom-update -d -f /lib/firmware/raspberrypi/bootloader/beta/pieeprom-2020-05-15.bin
+
+# Reboot and check the bootloader version 
+vcgencmd bootloader_version 
+vcgencmd bootloader_config
+```
+
+### Create a bootable USB drive
+* Use the [Raspberry Pi Imager](https://www.raspberrypi.org/downloads/) to flash Raspbian to a USB mass storage device. Other operating have not been tested and may require updates to u-boot. One reason for having a public beta is to help get USB MSD boot support into other distros.
+* Download the updated firmware files ``*.elf *.dat`` from the ``msd-boot`` branch of the [Raspberry Pi Firmware](https://github.com/raspberrypi/firmware) Github repo. 
+* Alternatively use ``rpi-update branch=msd-boot`` to update the firmware on a Raspbian SD card install and copy the files from there.
+* Copy these updates to the boot partition on the USB device. From now on ``rpi-update branch=msd-boot`` can be used from within Raspbian on the USB boot device.
+
+### USB device compatiblity
+This no explicit set of supported or not supported devices. Initially we recommend using a USB pen drive os SSD. Hard drives will probably require a powered HUB and in all cases you should verify that the devices work correctly from within Raspbian from an SD card boot first.
+
```

Comment:
os->or ?

Reply:
Not sure but perhaps "from an SD card boot" to "using an SD card boot" ?

Reply:
"This no explicit set of supported or not supported devices."

should probably be something like:

"There is no explicit set of supported devices."

I would remove the "and" between HUB and "in all cases" and make that two separate sentences, as it is a bit long the way it is. Also, it makes mores sense to have the bit about "all cases" as a sentence separate from the hard disk-specific stuff.